input/posts/en/transfer_learning.md

```diff
@@ -0,0 +1,37 @@
+﻿title: Transfer learning
+lead: Convolutional neural network training (CNN)
+Published: 2018-12-31
+Tags: [matlab, transfer learning]
+prerequisites: [Matlab]
+Authors: [tesar-tech, Karina9510]
+---
+This script carry out  a convolutional neural network training (CNN). The training of image classification problems is provided by the input image data stores, the network architecture layers, and the training options. Validation accuracy of the method is 100%.
```

Comment:
Too lengthy. Add information about created db with link to previous post. It has to make sense somehow... Also change information about accuracy from "100 %" to "almost 100 %". It depends on training data..